README.md

```diff
@@ -6,6 +6,45 @@ CNCF Storage Working Group
 
 Explore cloud native storage technology and concepts.
 
+## Why is storage important to cloud native
+
+- There’s no such thing as a stateless architecture, applications store state somewhere
+- Storage further enables specific cloud native patterns
+- Interoperating with storage increases cloud native’s relevance and leads to better applications
+
+## Mission statement
+
+In support of cloud native computing, the SWG intends to enable a thriving storage eco-system that is vendor and platform neutral and interoperable for applications
+
+## Responsibilities
+
+- Make sense of storage eco-system supporting cloud native environments
+- Review storage projects
+- Assist storage projects in TOC project nomination process
+
+## Governance
+
+The storage working group is an open group of community members with a leading representative as defined by the CNCF TOC (Ben Hindman). It meets regularly and members provide input and opinions about storage topics. Leadership makes decisions on SWG perspectives to be delivered to the TOC. 
+
+## What is Cloud Native Storage
+Cloud Native Storage is a classification for storage resources that are interoperable for applications in cloud native environments. It is not necessarily a cloud native system itself, but supports decoupling of storage services from applications. Interoperability is possible through standard storage interfaces. These resources are reliable, consumed on-demand, and self-provisioned. Resources can be elastic in both capacity and performance and utilization is measured to enable optimal and efficient use. The consumption experience is completely seamless where operations are transparent. 
```

Comment:
nit: 
"...both capacity and performance and utilization is measured" 
might be clearer if it were:
"...both capacity and performance utilization is measured"
(remove the second 'and')

Also, this sounds a bit too strong in that most services will require some initial setup before they become seamless:
"The consumption experience is completely seamless where operations are transparent. "

Reply:
Would you consider the initial setup part of consumption? The context is set early on to be from the perspective of applications.

Reply:
> Would you consider the initial setup part of consumption?
Yes.  Maybe that's the misunderstanding.

>The context is set early on to be from the perspective of applications.
I didn't get that the first time I read it.  I kind of see it now, but it isn't all that clear.  I worry that this definition could paint us into a corner - "Service XYZ requires some setup and configuration, therefore it isn't completely seamless".

Reply:
See updated text in the next commit, "The consumption experience for applications is seamless and transparent"